services/controller/standard/service.go

```diff
@@ -558,6 +563,33 @@ func (s *Service) handleBellatrixForkEpoch(ctx context.Context) {
 	}()
 }
 
+// fetchElectraForkEpoch fetches the epoch for the electra hard fork.
```

Comment:
Should consider replacing this with the function added in chaintime to obtain the hard fork, although I recognise that it's a copy of the other similar functions already in the controller.